addons/website_event/tests/test_website_event.py

```diff
@@ -58,6 +59,41 @@ def test_canonical_url(self):
 
 @tagged('post_install', '-at_install')
 class TestWebsiteAccess(HttpCaseWithUserDemo, TestWebsiteEventCommon):
+    @staticmethod
+    def are_names_in_order(names_in_expected_order, text):
+        last_index = -1
+        if isinstance(text, bytes):
+            text = text.decode('utf-8')
+        for name in names_in_expected_order:
+            current_index = text.find(name, last_index + 1)
+            if current_index == -1:
+                if text.find(name) != -1:
+                    return False
+                else:
+                    raise Exception(f"I can't check ordering when name: '{name}' is not present in the text.")
+            elif current_index <= last_index:
+                return False
+            last_index = current_index
+        return True
+
+    @freeze_time('2024-09-1')
+    def test_event_ordering(self):
```

Comment:
Should be in alphabetical order, and below the setup method

But also, this isn't the right class for this. I don't think we have UI tests, so it would need to be a new class of tests if anything.